frontend/src/helpers/Time.ts

```diff
@@ -0,0 +1,5 @@
+import { date } from 'quasar';
+
+export const useFormattedTimeStamp = (): string => {
```

Comment:
Good! Couple of remarks:

1. Let's refactor this function so it accepts `Date` parameter, and at call sites pass `Date.now()` as a param to it.  That way we can use it in the future for formatting other dates.
2. Maybe rename the function to better describe it's function i.e `formatDate` or `formatTimestamp` ?